tox.ini

```diff
@@ -59,3 +59,9 @@ deps =
     mypy
 commands =
     mypy --install-types --non-interactive .
+
+[testenv:update-charm-lib]
```

Comment:
**issue (blocking):** 

The command indirect result is the creation of an additional Python environment within .tox solely for its execution. If storage for the command is needed, a Makefile would be a more appropriate choice

Reply:
A script like previously will be fine (e.g. `scripts/update-charm-lib.sh`), using makefile for just a single target is overkill.

Reply:
unless you have more targets in mind?

Reply:
we won't need scripts for update dashboard files from openstack-exporter-k8s, it's very simple commands, can be in the workflow itself

Reply:
that being said, I don't oppose plan for the future

Reply:
Sure

Reply:
well, we can make it independent though, we create `scripts/*.sh` and let makefile to call the scripts

Reply:
> well, we can make it independent though, we create scripts/*.sh and let makefile to call the scripts

This one or makefile are work for me.